Assets/MixedRealityToolkit.Extensions/SpectatorView/SpectatorViewArchitecture.md

```diff
@@ -0,0 +1,67 @@
+# Overview
+
+[SpectatorView](Scripts/SpectatorView.cs) is a multi-device experience that allows a user's HoloLens application to be viewed by additional devices at their own viewpoints. It offers functionality for unidirectional content synchronization (State Synchronization) and leverages spatial coordinates for scene alignment (Spatial Alignment). It can be used to enable a variety of filming experiences including documenting prototypes and keynote demos.
+
+## Application Flow
+
+### Pre-compilation

```

Comment:
I assume it's the case that two different projects (with different asset caches) can't connect / sync? May want to call that out explicitly.